src/main/java/com/browserstack/automate/ci/jenkins/BrowserStackReportForBuild.java

```diff
@@ -164,7 +179,16 @@ private JSONObject convertSessionToJsonObject(Session session) {
             sessionJSON.put(Constants.SessionInfo.DURATION, Tools.durationToHumanReadable(session.getDuration()));
         }
 
-        sessionJSON.put(Constants.SessionInfo.CREATED_AT, session.getCreatedAt());
+        try {
+            Date sessionCreatedAt = Tools.SESSION_DATE_FORMAT.parse(session.getCreatedAt());
+            sessionJSON.put(Constants.SessionInfo.CREATED_AT, sessionCreatedAt);
+
+            String createdAtReadable = String.format("%s %s",
+                    Tools.readableDateFormat.format(sessionCreatedAt), "UTC");
+            sessionJSON.put(Constants.SessionInfo.CREATED_AT_READABLE, createdAtReadable);
+        } catch (ParseException ignored) {
```

Comment:
Are we not logging exception somewhere? Just swallow?

Reply:
its not really required actually. Such case shouldn't happen if we have session's data already. Will add the logging...

Reply:
added the logging